src/protections/MentionFlood.ts

```diff
@@ -0,0 +1,153 @@
+/*
```

Comment:
(Making this a new thread, so it can easily get replied on)

@deepbluev7 in a room mentioned, it might make sense to also match mentions based on the default pushrule behaviour. So for me, "mtrnord" would also mention me.

Should that go into this PR, or should that be in a follow up PR? (Asking as I think it makes sense to have it, but the current wave seems to mainly use mxids?)

Reply:
Let's start with a simple PR, then expand in a followup.